### PR TITLE
Feature/admin user edit

### PR DESCRIPTION
**Pull request by fabienalbi**
image

An admin can edit the profile of an user and give or remove admin rights

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -1,10 +1,35 @@
 class Admin::UsersController < AdminController
+  before_action :assign_user, only: %i(show edit update)
```
Comment (arnlen) [inline app/controllers/admin/users_controller.rb]: Nice one. 👍

File: spec/features/admin/user_edit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature 'User edit' do
+  let(:user) { create(:user) }
```
Comment (arnlen) [inline spec/features/admin/user_edit_spec.rb]: Prefer `given` instead of `let` for feature specs. ;)

File: spec/features/admin/user_edit_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  let(:admin) { build(:user, :admin) }
+
+  background do
+    sign_in_user(admin)
+    visit "/admin/users/#{user.id}"
+
+    click_on "Modifier l'utilisateur"
+
+    fill_in 'Nom', with: 'Kerbal'
```
Comment (arnlen) [inline spec/features/admin/user_edit_spec.rb]: lol

**Comment by arnlen** [comment]
Good job, 1 little cosmetic change and it will be good to merge. 👍

File: spec/features/admin/user_edit_spec.rb (hunk 2 above)
Comment (fabienalbi) [inline spec/features/admin/user_edit_spec.rb]: It is coming shortly :) There will be a PR dedicated to this made by Semia

**Pull request merged**